debezium-connector-mysql/src/main/java/io/debezium/connector/mysql/MySqlConnection.java

```diff
@@ -502,7 +492,22 @@ public MySqlConnectionConfiguration(Configuration config) {
             final Builder jdbcConfigBuilder = dbConfig
                     .edit()
                     .with("connectTimeout", Long.toString(getConnectionTimeout().toMillis()))
-                    .with("useSSL", Boolean.toString(useSSL));
+                    .with("sslMode", sslMode().name());
```

Comment:
```suggestion
                    .with("sslMode", sslMode().getValue());
```

I.e. let's not rely on the enum constant name, but the assigned value.